src/MeetingsSDKAdapter.js

```diff
@@ -315,12 +333,12 @@ export default class MeetingsSDKAdapter extends MeetingsAdapter {
       observer.complete();
     });
 
-    const muteEvent$ = fromEvent(sdkMeeting, EVENT_MEDIA_LOCAL_MUTE).pipe(
-      filter((event) => event.control === MUTE_VIDEO_CONTROL),
-      map(() => ({...muted, state: MeetingControlState.INACTIVE}))
+    const handleEvent$ = fromEvent(sdkMeeting, EVENT_MEDIA_LOCAL_UPDATE).pipe(
+      filter((event) => event.control === VIDEO_CONTROL),
+      map(({state}) => (state ? muted : unmuted))
     );
 
-    return concat(getDisplayData$, muteEvent$);
+    return concat(getDisplayData$, handleEvent$);
```

Comment:
Make sure that the default display is unmuted, as the control will start out inactive